src/main/resources/interface-specification.yml

```diff
@@ -1400,12 +1400,109 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/catalog/eservices/{eserviceId}/descriptor/{descriptorId}':
```

Comment:
I saw that for producer scope @galales uses the prefix` /producer` in the path.
Here I supposed that this endpoint is a catalog detail, but I don't have a strong opinion, I can easily switch to `/consumer`.
Please tell me your opinion.

Reply:
the current path works for me